config/index.mjs

```diff
@@ -1,5 +1,9 @@
 export const PORT = 5000;
-export const DB_CONNECTION = 'mongodb://localhost:27017';
+
+const { NODE_ENV: env } = process.env;
+
+export const DB_CONNECTION = `mongodb://${env === 'development' ? 'localhost' : 'mongo'}:27017`;
```

Comment:
Testing for the environment value is rarely flexible enough. Since we're going to have a bunch of values here and possibly more than two environments, a better approach would be to have ENV values per each key and provide default values here if ENV values are not set.

https://www.npmjs.com/package/convict is a great package which enforces you to document the used variables and supports the proposed approach.